libs/core/src/lib/declarations/known-keys.d.ts

```diff
@@ -16,6 +16,6 @@ type KnownKeysOfOptions = KnownKeys<Options>; // 'key' | 'title';
 
  * Taken from https://stackoverflow.com/questions/51465182/typescript-remove-index-signature-using-mapped-types
  */
-type KnownKeys<T> = {
+export type KnownKeys<T> = {
```

Comment:
Was this giving you issues?
I've been meaning to fix this - all of these declarations files should really be changed to just `.ts` files (and not `.d.ts`), and `export` from there.

Doing it as declaration files was an oversight of mine.